framework/audio/devtools/audioenginedevtools.cpp

```diff
@@ -116,7 +116,7 @@ void AudioEngineDevTools::makeArpeggio()
                          chunk.beginTick = tick;
                          chunk.endTick = chunk.beginTick + duration;
 
-                         uint32_t note_duration = duration / notes.size();
+                         uint32_t note_duration = uint32_t(duration / notes.size());
```

Comment:
please, do not use old style type conversion, use static_cast

Reply:
done